src/packages/v4/views/V4ProjectDashboard/V4ProjectTabs/V4TokensPanel/V4DistributeReservedTokensModal.tsx

```diff
@@ -0,0 +1,122 @@
+import { t, Trans } from '@lingui/macro'
+import TransactionModal from 'components/modals/TransactionModal'
+import useNameOfERC20 from 'hooks/ERC20/useNameOfERC20'
+import { useJBContractContext, useReadJbTokensTokenOf, useReadJbTokensTotalCreditSupplyOf } from 'juice-sdk-react'
+import { useDistributeReservedTokens } from 'packages/v2v3/hooks/transactor/useDistributeReservedTokensTx'
```

Comment:
Needs a nuke i think?

Honestly at this stage, i would completely not worry about any transaction-related features. I can take them 🙏 , need to establish a Wagmi way of doing txs first (and that work with the modals)

Reply:
Got it 👍